windows_install_build_dependencies.bat

```diff
@@ -122,57 +122,57 @@ IF /I EXIST %bash_path% (
 ::::::::::::::::::::::: Install in c:\PR-XXXX\ section started - once per build, reinstalled only with expunge clean ::::::::::::::::::::::::::::::::::
 :::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
-
 :::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
-::::::::::::::::::::::: OpenVINO - reinstalled per build trigger
-set "openvino_dir=openvino_toolkit_windows_2025.1.0.dev20250214_x86_64"
-set "openvino_ver=openvino_toolkit_windows_2025.1.0.dev20250214_x86_64.zip"
-set "openvino_http=https://storage.openvinotoolkit.org/repositories/openvino/packages/nightly/2025.1.0-18174-6f7519ad4dc/"
+::::::::::::::::::::::: GENAI/OPENVINO - reinstalled per build trigger
+set "genai_dir=openvino_genai_windows_2025.1.0.0.dev20250221_x86_64"
+set "genai_ver=openvino_genai_windows_2025.1.0.0.dev20250221_x86_64.zip"
```

Comment:
This will create mismatch with Linux version. Do we want to install newer version for windows or the change for linux will follow?